apps/docs/src/development/i18n.md

```diff
@@ -47,7 +47,7 @@ import deDE from "./i18n/locales/de-DE.json";
 const i18n = createI18n({
   legacy: false,
   locale: "en-US",
-  messages: { "en-US": enUS, "de-DE": deDE },
+  messages: { "en-US": enUS, "de-DE": deDE, "ko-KR": koKR },
```

Comment:
This line is the example for vue-i18n which is your project individual translations.

The onyx translations need to be passed to the `createOnyx()` function.

Reply:
Yes, Indeed. I tested it by running `alpha-test-app`.